box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxFolder.java

```diff
@@ -118,7 +118,10 @@ public BoxUploadEmail getUploadEmail() {
      * @return true if the folder has collaborations; otherwise false.
      */
     public Boolean getHasCollaborations() {
-        return (Boolean) mProperties.get(FIELD_HAS_COLLABORATIONS);
+        Object hasCollaborators = mProperties.get(FIELD_HAS_COLLABORATIONS);
+        if (hasCollaborators != null)
+            return (Boolean) hasCollaborators;
+        return false;
```

Comment:
This should actually return null as that is actually a valid value that the server can return (ie. property not set)